src/style.css

```diff
@@ -0,0 +1,105 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+.input {
+  width: 100%;
+  height: 100%;
+}
+
+.input-big {
+  color: #3d4e61;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  height: 70px;
+  padding: 25px 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.input-small {
+  color: #3d4e61;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  height: 42px;
+  padding: 12px 33px;
+  font-size: 14px;
+  line-height: 19px;
+}
```

Comment:
pay attention to point 2 of [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)
you have a lot of repeated styles - fix for another elements